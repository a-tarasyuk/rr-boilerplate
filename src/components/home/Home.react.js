import './home.css';
import React from 'react';

export default () => (
  <div className="home">
    <h1 className="home__title">rr-boilerplate</h1>
    <p className="home__description">Lightweight React&Redux boilerplate</p>
  </div>
);