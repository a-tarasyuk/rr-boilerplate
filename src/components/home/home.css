.home {
  &__title {
    margin: 0;
    padding: 0;
    font-size: 1.8em;
  }

  &__description {
    margin: 0;
    padding: 0;
    color: #777777;
    font-size: 1em;
  }
}