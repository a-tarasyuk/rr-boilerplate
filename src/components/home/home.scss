@import '~styles/theme/variables';

.home {
  &__title {
    margin: 0;
    padding: 0;
    font-size: 1.8em;
  }

  &__description {
    margin: 0;
    padding: 0;
    color: $boulder;
    font-size: 1em;
  }
}