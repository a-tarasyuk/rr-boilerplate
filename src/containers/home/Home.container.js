import React from 'react';
import { connect } from 'react-redux';
import Home from 'components/home/Home.react';

const HomeContainer = () => (
  <Home />
);

export default connect()(HomeContainer);