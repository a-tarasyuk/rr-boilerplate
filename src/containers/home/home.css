.home {
}

.title {
  margin: 0;
  padding: 0;
  font-size: 1.8em;
}

.description {
  margin: 0;
  padding: 0;
  color: #777;
  font-size: 1em;
}
