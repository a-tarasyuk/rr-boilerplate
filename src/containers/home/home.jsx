import React from 'react';
import styles from './home.css';

export const Home = () => (
  <div className={ styles.home }>
    <h1 className={ styles.title }>rr-boilerplate</h1>
    <p className={ styles.description }>Lightweight React&Redux boilerplate</p>
  </div>
);

export default Home;
