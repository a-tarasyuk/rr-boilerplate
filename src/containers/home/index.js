import styles from './styles.css';
import React from 'react';

const Home = () => {
  return <div className={ styles.home }>
    <h1 className={ styles.title }>rr-boilerplate</h1>
    <p className={ styles.description }>Lightweight React&Redux boilerplate</p>
  </div>;
};

export default Home;