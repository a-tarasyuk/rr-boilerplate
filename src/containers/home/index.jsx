import React from 'react';
import styles from './styles.css';

const Home = () => (
  <div className={styles.home}>
    <h1 className={styles.title}>rr-boilerplate</h1>
    <p className={styles.description}>Lightweight React&Redux boilerplate</p>
  </div>
);

export default Home;
