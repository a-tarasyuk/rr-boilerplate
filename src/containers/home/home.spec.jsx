import React from 'react';
import { shallow } from 'enzyme';
import { Home } from './home';

describe('<Home />', () => {
  it('matches its snapshot', () => {
    const component = shallow(<Home />);
    expect(component).toMatchSnapshot();
  });
});
