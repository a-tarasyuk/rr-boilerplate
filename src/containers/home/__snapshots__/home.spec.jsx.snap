// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> matches its snapshot 1`] = `
<div
  className="home"
>
  <h1
    className="title"
  >
    rr-boilerplate
  </h1>
  <p
    className="description"
  >
    Lightweight React&Redux boilerplate
  </p>
</div>
`;
